src/functions/lever.js

```diff
@@ -0,0 +1,67 @@
+const got = require('got')
+const { parse, format } = require('url')
+const isEqual = require('lodash.isequal')
+
+/**
+ *
+ * This lambda is used to check for any differences in roles listed on yld.io
+ * and our lever account. If any new changes are found we trigger a new site deployment.
+ *
+ * The yld.io/meta.json endpoint is created on the postBuild method of gatsby.
+ * See gatsby-node.js for more detials.
+ *
+ * I've not found a way to test this in a deploy preview as we have no reference to the
+ * deploy url e.g. "https://deploy-preview-203--yldio.netlify.com/.netlify/functions/lever"
```

Comment:
Did @omonk not just demo-ed `https://deploy-preview-203--yldio.netlify.com/meta.json` to us? I'm a bit confused :confused:

Reply:
Ah this line is probably a bit confusing actually. I was referring to the fact that the function doesn't know the context its in e.g. `production` /`deploy-preview` / `branch-deploy`. Functions aren't given any sort of env variable to let them know :/

Reply:
so when I mentioned `https://deploy-preview-203--yldio.netlify.com/.netlify/functions/lever` - We know where it is but when the function runs on the aws box, it doesn't know whether its on yld.io or a deploy-previwe